db/schema.rb

```diff
@@ -0,0 +1,58 @@
+# encoding: UTF-8
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20160427220429) do
+
+  create_table "markets", force: :cascade do |t|
+    t.integer  "market_id"
+    t.text     "name",       null: false
+    t.text     "address"
+    t.text     "city"
+    t.text     "county"
+    t.text     "state"
+    t.integer  "zip_code"
```

Comment:
Zip code should probably be a string type as well since no math is being done with it. Also, if a zip code happened to start with a 0, then the leading 0 would be removed with the integer type.